src/logging.js

```diff
@@ -29,49 +29,44 @@ const prodFormatter = ({message, durationMs}: Info): string =>
 const devFormatter = (info: Info): string =>
     `${info.level}: ${prodFormatter(info)}`;
 
-function getFormatters(json, isDev) {
+function getFormatters(isDev) {
     const formatters: Array<Format> = [
         winston.format.splat(), // Allows for %s style substitutions
     ];
 
-    if (json) {
-        formatters.push(winston.format.json());
-        if (isDev) {
-            formatters.push(winston.format.prettyPrint({colorize: true}));
-        }
```

Comment:
No more JSON. Doesn't make sense and looks pretty bad when we see errors and it was only used for errors.